seekr/console_scripts.py

```diff
@@ -400,8 +406,17 @@ def console_kmer_counts():
                         help='Set if output should not have the mean subtracted.')
     parser.add_argument('-us', '--unstandardized', action='store_false',
                         help='Set if output should not be divided by the standard deviation.')
-    parser.add_argument('-nl', '--no_log2', action='store_false',
-                        help='Set if output should not be log2 transformed.')
+
+    '''Dan S. Changes
+    Add arguments for log transforming pre standardize, log transforming post standardize, or no log
+    Check to see that only 1 of the three arguments is passed
+    If none passed, stop program
+    '''
+    parser.add_argument('-l', '--log2', type=int,default=2,
```

Comment:
Looks like we need https://docs.python.org/3/library/argparse.html#choices

Again, would go for `['pre', 'post', 'none']` here.